### PR TITLE
mare-finance emissions adapter

### PR DESCRIPTION
**Pull request by Define101**
mare-finance emissions adapter

### CODE REVIEW

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/mare-finance.ts" should be below: 
 
 emissions schedule

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/mare-finance.ts" should be below: 
 
 emissions schedule

File: protocols/mare-finance.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const start_rewards = "1679918400"
+
+
+const mare: Protocol = {
+"Initial Liquidity": manualCliff(start, qty * 0.025),
+"LFG": manualLinear(start, start + periodToSeconds.month * 12, qty * 0.035),
+"Airdrop": manualCliff(start, qty * 0.03),
+"Sonne Stakers": manualCliff(start, qty * 0.15),
+"Protocol Growth": manualCliff(start, qty * 0.04),
+"Core Team": manualLinear(start, start + periodToSeconds.year * 2, qty * 0.12),
```
Comment (waynebruce0x) [inline protocols/mare-finance.ts:16]: missing the 3mo cliff here

File: protocols/mare-finance.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+"Rewards & Bribes":manualLinear(start, start + periodToSeconds.month * 36, qty * 0.6),
```
Comment (waynebruce0x) [inline protocols/mare-finance.ts:17]: I did a table (like the one shown in mare docs) in the TraderJoe adapter which could help

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/mare-finance.ts" should be below: 
 
 emissions schedule

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/mare-finance.ts" should be below: 
 
 emissions schedule

**Pull request closed without merging**